_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,412 @@
+
+```html
+<head>
+	<title>Passed example 5 9bd38c</title>
+	<style>
+		div.tilt {
+			height: 750px;
+			width: 150px;
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+		}
+	</style>
+</head>
+<body>
+	<div class="tilt">look at this pieCe of tiLted text fOr clueS on whEre to find The monster.</div>
+</body>
+```
+
+#### Passed Example 6
+
+The button is indicated by the word "round". But the word is also included in the text of the element.
+
+```html
+<head>
+	<title>Passed example 6 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 7
+
+The images are indicated by the visual indicator words "narrow" and "wide". These words are also included in the accessible names of the images. Even through that indication is not visible, it is included in the accessibility tree which is sufficient to pass this rule.
+
+```html
+<p>The wide image is awesome. But the narrow image isn't.</p>
+<img scr="/test-assets/images/awesome_wide.jfif" alt="Wide photo of an awesome landscape." />
+<img scr="/test-assets/images/Non_awesome_narrow.jpg" alt="Narrow photo of a dull landscape." />
+```
+
+#### Passed Example 8
+
+This document is using "triangle" which is included in the heading of the menu; the fact that the described content is one another page does not restrict this rule.
+
+```html
+<p>
+	On the <a href="/test-assets/SC1.3.3-triangle-menu-with-heading.html">information page</a> you can find more examples
+	within the triangle menu
+</p>
+```
+
+#### Passed Example 9
+
+This document is using "star" but there is also a heading "examples" that can be referenced; the fact that the described content is in an iframe does not restrict this rule.
+
+```html
+<p>More examples can be found when you look underneath the star or you can search for the "Examples" heading</p>
+<iframe src="/test-assets/SC1.3.3-star-with-heading.html"></iframe>
+```
+
+#### Passed Example 10
+
+This document is using the word "circle" but in this case it is no instruction so it passes.
+
+```html
+<p>
+	This circle is nice.
+	<span style="height: 25px;
+		     width: 25px;
+		     background-color: #bbb;
+		     border-radius: 50%;
+		     display: inline-block;">
+	</span>
+</p>
+```
+
+#### Passed Example 11
+
+This document is using the word "circle" but in this case it is no instruction so it passes. Note that this example is not inapplicable despite the fact that the text is not visible.
```

Comment:
```suggestion
This document is using the word "circle" but in this case it is no instruction so it passes. Note that this example is not inapplicable because, despite the fact that the text is not visible, it is included in the accessibility tree.
```